ontology/yaml/resources/fields/telemetry_fields.yaml

```diff
@@ -843,6 +843,9 @@ literals:
 - supply_air_flowrate_status:
   - ON
   - OFF
+- low_water_alarm:
```

Comment:
we have `low_level_status` already, could we use this as precedent for this alarm by calling it `low_level_alarm`?

Reply:
Yes sure